src/Text/Pandoc/Readers/LaTeX.hs

```diff
@@ -726,6 +728,14 @@ dolstinline = do
 doLHSverb :: PandocMonad m => LP m Inlines
 doLHSverb = codeWith ("",["haskell"],[]) <$> manyTill (satisfy (/='\n')) (char '|')
 
+dosiunitx :: PandocMonad m => LP m Inlines
+dosiunitx = do
+  skipopts
+  value <- char '{' >> manyTill anyChar (char '}')
+  preunit <- (char '[' >> manyTill anyChar (char ']')) <|> (return "")
```

Comment:
I use the option command now.

`bracketed` does not parse latex commands e.g. `\$`. So I cannot write `\SI{1.5}[\$]{}`.